### PR TITLE
Feat/st 169 checkout page

### PR DESCRIPTION
**Pull request by HoThiSang**

### CODE REVIEW

File: src/pages/users/Checkout.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+               <option value="">--Choose an option--</option>
+                <option value="VNP">VNP</option>
+                <option value="COD">COD</option>
+              </select>
+            </div>
+            <hr
+              style={{
+                borderWidth: "2px",
+                borderColor: "#C4C4C4",
+                marginTop: "10px"
```
Comment (hongphong88) [inline src/pages/users/Checkout.js]: Don't use style inline. Create css class for this one

File: src/pages/users/Checkout.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+              <input type="hidden" name="total_price" />
+            </div>
+          </div>
+          <div className="form-group col-md-12 text-right">
+            <div
+              className="form-inline"
+              style={{
+                display: "flex",
+                justifyContent: "space-between",
+                alignItems: "center"
```
Comment (hongphong88) [inline src/pages/users/Checkout.js]: Don't use style inline. Create css class for this one

File: src/pages/users/Checkout.js [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+            <hr
+              style={{
+                borderWidth: "2px",
+                borderColor: "#C4C4C4",
+                marginTop: "10px"
+              }}
+            />
+            <div
+              className="form-inline"
+              style={{
```
Comment (hongphong88) [inline src/pages/users/Checkout.js]: Don't use style inline. Create css class for this one

File: src/pages/users/Checkout.js [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+                </div>
+              </>
+            ))}
+          </div>
+        </div>
+        <div className="form-container">
+          <div className="form-group col-md-12 text-right">
+            <hr
+              style={{
+                borderWidth: "2px",
```
Comment (hongphong88) [inline src/pages/users/Checkout.js]: Don't use style inline. Create css class for this one

File: src/pages/users/Checkout.js [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+            <div className="form-group col-md-3">
+              <label htmlFor="quantity">Quantity</label>
+            </div>
+            <div className="form-group col-md-3">
+              <label htmlFor="unit-price">Unit price</label>
+            </div>
+            <div className="form-group col-md-3">
+              <label htmlFor="total-price">Total price</label>
+            </div>
+            {carts.map((item, index) => (
```
Comment (hongphong88) [inline src/pages/users/Checkout.js:134]: Create new component for this item

File: src/pages/users/Checkout.js [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+    <form className="form-checkout" onSubmit={handelSubmitForm}>
+      <div className="container">
+        <h3 className="checkout-heading">CHECKOUT PAGE</h3>
+        <hr style={{ borderWidth: "2px", borderColor: "#C4C4C4" }} />
+        <h4 className="checkout-heading text-center mt-4">Bill information</h4>
+
+        <div className="row">
+          <div className="form-container">
+            
+            <div className="form-group mb-3 col-md-6">
```
Comment (hongphong88) [inline src/pages/users/Checkout.js]: Create new component for this one => 
 `<div className="form-group mb-3 col-md-6">
              <label htmlFor="" className="form-label">Email</label>
              <input type="email" className="form-control" value={email}  onChange={(e) =>setEmail(e.target.value)} />
            </div>`

File: src/components/BannerBottom.js [hunk 7]
```diff
@@ -0,0 +1,49 @@
+const BannerBottom = () => {
+    return (
+        <div className="info-boxes wow fadeInUp">
+        <div className="info-boxes-inner">
+          <div className="row">
+            <div className="col-md-6 col-sm-4 col-lg-4">
```
Comment (hongphong88) [inline src/components/BannerBottom.js]: Create new component => `<div className="col-md-6 col-sm-4 col-lg-4">
              <div className="info-box">
                <div className="row">
                  <div className="col-xs-12">
                    <h4 className="info-box-heading green">
                      money back
                    </h4>
                  </div>
                </div>
                <h6 className="text">30 Days Money Back Guarantee</h6>
              </div>
            </div>`

File: src/pages/users/Checkout.js (hunk 5 above)
Comment (HoThiSang) [reply src/pages/users/Checkout.js:134]: I have just resolved

File: src/pages/users/Checkout.js (hunk 6 above)
Comment (HoThiSang) [reply src/pages/users/Checkout.js]: I have just resolved

File: src/components/BannerBottom.js (hunk 7 above)
Comment (HoThiSang) [reply src/components/BannerBottom.js]: I have just resolved

File: src/pages/users/Checkout.js (hunk 3 above)
Comment (HoThiSang) [reply src/pages/users/Checkout.js]: I have just resolved

File: src/pages/users/Checkout.js (hunk 1 above)
Comment (HoThiSang) [reply src/pages/users/Checkout.js]: I have just resolved

**Pull request merged**